code/espurna/thinkspeak.ino

```diff
@@ -96,6 +103,10 @@ void _tspkConfigure() {
         setSetting("tspkEnabled", 0);
     }
     if (_tspk_enabled && !_tspk_client) _tspkInitClient();
+  
+    _tspk_host = getSetting("tspkHost", THINGSPEAK_HOST);
+    _tspk_url = getSetting("tspkUrl", THINGSPEAK_URL);
+    _tspk_port = getSetting("tspkPort", THINGSPEAK_PORT);
```

Comment:
Small issue with async client is that it is *possible* to have different host value in `_tspkPost()` and `_tspk_client->onConnect()`. `tspkPost` is expected to be called only once per request, so it would be safer to load things there?

Reply:
You're clearly right. I'm agree with it. I will do it asap. Thanks.

Reply:
You're clearly right. I'm agree with it. I pushed a new version.